spacepy/empiricals.py

```diff
@@ -670,7 +672,7 @@ def omniFromDirectionalFlux(fluxarr, alphas, norm=True):
        fac = 2*np.pi
        denomina = 1
     alphrad = np.deg2rad(alphas)
-    numera = integ.simps(fluxarr*np.sin(alphrad), alphrad)
+    numera = integ.simpson(fluxarr*np.sin(alphrad), alphrad)
```

Comment:
Looks like [x must be a kwarg](https://docs.scipy.org/doc/scipy/reference/generated/scipy.integrate.simpson.html) but [it's been named x even in simps](https://docs.scipy.org/doc/scipy-0.9.0/reference/generated/scipy.integrate.simps.html) so should be fine to just make it a kwarg here and it'll still work on the old code.